scaling_performance/network_optimization.adoc

```diff
@@ -0,0 +1,76 @@
+[[scaling-performance-network-optimization]]
+= Network Optimization
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+[[scaling-performance-network-performance]]
+== Optimizing Network Performance
+
+The xref:../architecture/additional_concepts.adoc#openshift-sdn[OpenShift SDN] uses OpenvSwitch, VXLAN tunnels, OpenFlow rules, and
+iptables. This network can be tuned by using jumbo frames, NIC offloads,
+multi-queue, and ethtool settings.
+
+VXLAN provides benefits over VLANs, such as an increase in networks from 4096 to
+over 16 million, and layer 2 connectivity across physical networks. This allows
+for all pods behind a service to communicate with each other, even if they are
+running on different systems.
+
+However, this leads to increased CPU utilization. VXLAN encapsulates all
+tunneled traffic in UDP packets. Both these outer- and inner-packets are subject
+to normal checksumming rules to guarantee data has not been corrupted during
+transit. Depending on CPU performance, this additional processing overhead can
+cause a reduction in throughput and increased latency when compared to
+traditional, non-Overlay networks.
+
+Cloud, VM, and bare metal CPU performance can be capable of handling well over
```

Comment:
s/can be capable of handling well over one Gbps/can handle much more than one Gbps